### PR TITLE
add news for Firmware Bugfix

### PR DESCRIPTION
**Pull request by T0biii**
add news for Firmware Bugfix

image

### CODE REVIEW

**Review by awlx — APPROVED** [review]

File: _posts/2024-03-17-firmware-update.markdown [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+layout: post
+title:  "Firmware-Update: Behebung von Wiederverbindungsproblemen"
+date:   2024-03-17 20:00:00
+categories: freifunkmuc
+---
+
+## Firmware-Update v2024.1.5: Behebung von Wiederverbindungsproblemen
+
+Während unserer Wartungsarbeiten ist uns ein Fehler aufgefallen, der vor allem beim Abschalten eines Gateways auftritt.
```
Comment (krombel) [inline _posts/2024-03-17-firmware-update.markdown:10]:
```suggestion
Während unserer Wartungsarbeiten ist uns ein Fehler aufgefallen, der vor allem beim Abschalten eines Gateways auftritt:
```

File: _posts/2024-03-17-firmware-update.markdown [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+layout: post
+title:  "Firmware-Update: Behebung von Wiederverbindungsproblemen"
+date:   2024-03-17 20:00:00
+categories: freifunkmuc
+---
+
+## Firmware-Update v2024.1.5: Behebung von Wiederverbindungsproblemen
+
+Während unserer Wartungsarbeiten ist uns ein Fehler aufgefallen, der vor allem beim Abschalten eines Gateways auftritt.  
+Das Wiederverbinden eines Nodes dauerte bis zu 30 Minuten, was nun jedoch auf unter 2 Minuten verbessert wurde.
```
Comment (krombel) [inline _posts/2024-03-17-firmware-update.markdown:11]:
```suggestion
Das Wiederverbinden eines Nodes dauerte bis zu 30 Minuten.
```

File: _posts/2024-03-17-firmware-update.markdown [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+Deshalb veröffentlichen wir bald die Version v2024.1.5, um das Problem zu beheben
+
+## Lösung
+
+Wir haben intensive Änderungen vorgenommen, um das Wiederverbindungsverhalten zu verbessern.
```
Comment (krombel) [inline _posts/2024-03-17-firmware-update.markdown:16]:
```suggestion
Wir haben intensive Änderungen vorgenommen, um das Wiederverbindungsverhalten zu verbessern. Das Wiederverbinden eines Nodes dauert nun bis zu 2 Minuten.
```

**Review by krombel — COMMENTED** [review]
Just a few changes **_I_** would do

// EDIT: Seems I came to late. You might ignore it

**Comment by T0biii** [comment]
@krombel https://github.com/freifunkMUC/freifunkmuc.github.io/pull/400 <3

**Pull request merged**